components/menus/destinations.ts

```diff
@@ -149,8 +159,44 @@ export function destinationsMenu(req: RequestWithJwt): GameFacingDestination[] {
                     req.jwt.unique_name,
                     req.gameVersion,
                 ),
+                Data:
+                    req.gameVersion === "h1"
+                        ? {
+                              normal: {
+                                  ChallengeCompletion: undefined,
+                                  CompletionData: generateCompletionData(
+                                      destination,
+                                      req.jwt.unique_name,
+                                      req.gameVersion,
+                                      "mission",
+                                      "normal",
+                                  ),
+                              },
+                              pro1: {
+                                  ChallengeCompletion: undefined,
+                                  CompletionData: generateCompletionData(
+                                      destination,
+                                      req.jwt.unique_name,
+                                      req.gameVersion,
+                                      "mission",
+                                      "pro1",
+                                  ),
+                              },
+                          }
+                        : undefined,
             },
         }
+
+        // TODO: THIS IS NOT CORRECT FOR 2016!
```

Comment:
Should we make a new issue in order to not forget?

Reply:
Sure.